<hr>

# Module 19.1 - The Foreign Exchange Market

<hr>

<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/01.png?raw=true">

### LOS 19.a: Calculate and interpret currency cross-rates.

A cross rate is the exchange rate between two currencies implied by their exchange rates with a common third currency. Cross-rates are necessary when there is no active FX market in a currency pair. The rate must be computed from the exchange rates between each of these two currencies and a third currency, usually the USD or EUR.

Let's assume that we have the following quotations for Mexican pesos and Australian dollars: MXN/USD = 10.70 and USD/AUD = 0.60. The cross rate between Australian dollars and pesos (MXN/AUD) is as follows:

    MXN/AUD = USD/AUD × MXN/USD = 0.60 × 10.70 = 6.42
    
    
<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/02.png?raw=true">    

So, our MXN/AUD cross rate is 6.42 pesos per Australian dollar. The key to calculating cross-rates is to note that the basis of the quotations must be such that we get the desired result algebraically. If we had started with an AUD/USD quotation of 1.67, we would have taken the inverse to get the quotation into USD/AUD terms. Another approach is to divide through, as is illustrated in the following example.

In [39]:
### LOS 19.a: Calculate and interpret currency cross-rates.
### MXN/AUD = USD/AUD × MXN/USD
MXNAUDcross = 0.60 * 10.70

print("", round(MXNAUDcross,5), "MXN/AUD is the cross-rate between USD/AUD x MXN/USD ")

 6.42 MXN/AUD is the cross-rate between USD/AUD x MXN/USD 


#### Example: Cross rate calculation

The spot exchange rate between the Swiss franc (CHF) and the USD is CHF/USD = 1.7799, and the spot exchange rate between the New Zealand dollar (NZD) and the U.S. dollar is NZD/USD = 2.2529. Calculate the CHF/NZD spot rate.

#### Answer:

The CHF/NZD cross rate is (CHF/USD) / (NZD/USD) = 1.7799 / 2.2529 = 0.7900.

<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/03.png?raw=true">
<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/04.png?raw=true">
<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/05.png?raw=true">

In [38]:
### Example 2: Cross rate calculation
# The spot exchange rate between the Swiss franc (CHF) 
# and the USD is CHF/USD = 1.7799, and the spot exchange 
# rate between the New Zealand dollar (NZD) and the U.S. 
# dollar is NZD/USD = 2.2529. Calculate the CHF/NZD spot rate.

### Cross rate is (CHF/USD) divided by (NZD/USD)
CHFNZDcross = (1.7799 / 2.2529)
print("", round(CHFNZDcross,4), "CHF/NZD is the cross-rate (CHF/USD) divided by (NZD/USD)")

 0.79 CHF/NZD is the cross-rate (CHF/USD) divided by (NZD/USD)


### LOS 19.b: Explain the arbitrage relationship between spot and forward exchange rates and interest rates, calculate a forward rate using points or in percentage terms, and interpret a forward discount or premium.

When currencies are freely traded and forward currency contracts exist, the percentage difference between forward and spot exchange rates is approximately equal to the difference between the two countries' interest rates. This is because there is an arbitrage trade with a riskless profit to be made when this relation does not hold.

We call this a no-arbitrage condition because if it doesn't hold, there is an opportunity to make a profit without risk. The possible arbitrage is as follows: borrow Currency A at Interest Rate A, convert it to Currency B at the spot rate and invest it to earn Interest Rate B, and sell the proceeds from this investment forward at the forward rate to turn it back into Currency A. If the forward rate does not correctly reflect the difference between interest rates, such an arbitrage could generate a profit to the extent that the return from investing Currency B and converting it back to Currency A with a forward contract is greater than the cost of borrowing Currency A for the period.

The no-arbitrage condition requires that "you cannot earn more than your domestic riskless rate of interest by borrowing your domestic currency, converting it to a foreign currency to invest at the foreign riskless rate, and exchanging back to your domestic currency." So:

$(1+r_{\text{domestic}})= \Large\frac{1}{\text{spot}_{\text{d/f}}}(1+r_{\text{ foreign}})\text{ forward}_{\text{d/f}}$

The three terms on the right-hand side are equivalent to units of foreign currency for one unit of the domestic currency, growth if invested at the foreign interest rate, and then exchanged back to domestic currency at the forward rate.

Equivalent relationships are as follows:
\
\
$\Large \frac{\text{Forward}_{\text{d/f}}}{\text{spot}_\text{d/f}} = \frac{(1 + r_{\text{ domestic}})}{(1 + r_\text{ foreign})}$
\
\
$\quad\text{Forward}_{\text{d/f}} =  \Large\frac{(1 + r_{\text{ domestic}})}{(1 + r_\text{ foreign})} \small \text{spot}_\text{d/f} $
\
\
$\qquad\text{spt}_{\text{d/f}} =  \Large\frac{(1 + r_{\text{ domestic}})}{(1 + r_\text{ foreign})} \small \text{forward}_\text{d/f} $


#### Example: Calculating the arbitrage-free forward exchange rate

Consider two currencies, the ABE and the DUB. The spot `ABE/DUB` exchange rate is `4.5671`, the 1-year riskless `ABE` rate is `5%`, and the 1-year riskless `DUB` rate is `3%`. What is the 1-year no-arbitrage forward exchange rate?

#### Answer:
\
\
$ \text{forward}_\text{A/D} = \text{spot}_\text{A/D}$ 
$\large\Bigg(\frac{1+r_A}{1+r_D} \Bigg)$
\
\
$ \text{forward}_\text{A/D} = 4.5671_\text{A/D}$ 
$\large\Bigg(\frac{1.05}{1.03} \Bigg) = \small 4.6558_\text{A/D}$
\
\
\
Note that the forward rate is greater than the spot rate by `4.6558 / 4.5671 – 1 = 1.94%`. This is approximately equal to the interest rate differential of` 5% – 3% = 2%`. 


* The currency with the higher interest rate must depreciate over time by approximately the amount of the interest rate differential, to prevent arbitrage.

* The country with the *higher* interest rate, $r$, will see its currency *depreciate* (i.e., trade at a forward discount)

* The <FONT COLOR="#ff0000">no-arbitrage forward rate</FONT> is driven by the the differences in the two countries' interest rates.

* interest rate parity = no arbitrage here


In [58]:
FwdAD1 = round(4.5671 * (1.05/1.03),4)
print(FwdAD1, "is the forward rate of ABE/DUB for 1 year")

spotAD1 = ((4.6558 / 4.5671) - 1)*100
print("Note that the forward rate is greater than the spot rate by", round(spotAD1,2),"%")
print("Note that the forward rate approximation = 2% > 1.94%, arb oppurtunity?")
print("Often there is arbitrage oppurtunity and it's expoited here 0.060 %")

4.6558 is the forward rate of ABE/DUB for 1 year
Note that the forward rate is greater than the spot rate by 1.94 %
Note that the forward rate approximation = 2% > 1.94%, arb oppurtunity?
Often there is arbitrage oppurtunity and it's expoited here 0.060 %


### If we are calculating a 30-, 90-, or 180-day forward exchange rate, we need to use interest rates for 30-, 90-, and 180-day periods rather than annual rates. Note that these shorter-term rates are quoted as money market yields.



#### Example: Calculating the arbitrage-free forward exchange rate with 90-day interest rates

The spot `ABE/DUB` exchange rate is `4.5671`, the 90-day riskless `ABE` rate is `5%`, and the `90-day` riskless `DUB` rate is `3%`. 

What is the 90-day forward exchange rate that will prevent arbitrage profits?

#### Answer:

$\qquad\text{no-arbitrage forward} = 4.5671 \Large \Biggr[\frac{1+(0.05\frac{(90}{360)}}{1+(0.03\frac{(90}{360)}}\Biggr]$
$= 4.5671 \Large\Big(\frac{1.0125}{1.0075} \Big)= \small 4.5898 \text{  ABE/DUB}$


In [66]:
#### Example: Calculating the arbitrage-free forward exchange 
# rate with 90-day interest rates

# The spot ABE/DUB exchange rate is 4.5671, 
# the 90-day riskless ABE rate is 5%, 
# and the 90-day riskless DUB rate is 3%.


# What is the 90-day forward exchange rate that will 
# prevent arbitrage profits?

# Define the spot rate
ABEDUB = 4.5671

# Change from 1 yr 360 to 90 day adjustment
ABEDUBNoarb90 = (1+(.05*(90/360)))/(1+(.03*(90/360))) * ABEDUB 
 
print(round(ABEDUBNoarb90,4), "is the 90 day forward exchange rate that prevents no arb profits")

4.5898 is the 90 day forward exchange rate that prevents no arb profits


<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/08.png?raw=true">
<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/06.png?raw=true">
<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/07.png?raw=true">


#### No-Arbitrage and Interest Rate Parity

* if forward rates deviate from **no-arbitrage-condition**, an arbitrage profit may be earned.

* Arbitrageurs will pursue this oppurtunity until the no-arbitrage condition is restored.

* In some markets, frictions, restrictions, and regulations make arbritrage more diffucult to implement.

* country with the HIGHER interest rate in calc above would depreciate
* country with the LOWER interest rate in calc above would appreciate

* If forward rates did deviate from our no-arbitrage condition this idea of a carry trade becomes profitable.
    * If you could borrow cheaply & convert over to a higher interest rate part of the world
    * hold it for a period
    * then convert it back
    * If you can hang on to that profit that's called <mark style="color: green;">Carry Trade</mark>
    * Acdording to no arbitrage that won't hold for long, if did plenty of arbitrageurs would take advantage until any profit that existed was gone.  (No free lunch)

In our previous example, we calculated the no-arbitrage one-year forward ABE/DUB exchange rate as 4.6558. Next, we illustrate the arbitrage profit that could be gained if the forward exchange rate differs from this no-arbitrage rate. Consider a forward rate of 4.6000 so that the depreciation in the ABE is less than that implied by the no-arbitrage relationship. This makes the ABE attractive to a DUB investor who can earn a riskless profit as follows:

* Borrow 1,000 DUB for one year at 3% to purchase ABE and get 4,567.1 ABE.
* Invest the 4,567.1 ABE at the ABE rate of 5% to have 1.05(4,567.1) = 4,795.45 ABE at the end of one year.
* Enter into a currency forward contract to exchange 4,795.45 ABE in one year at the forward rate of 4.6000 ABE/DUB to receive 4,795.45 / 4.6000 = 1,042.49 DUB.

The investor has ended the year with a 4.249% return on his 1,000 DUB investment, which is higher than the 3% 1-year DUB interest rate. After repaying the 1,000 DUB loan plus interest (1,030 DUB), the investor has a profit of 1,042.49 – 1,030 = 12.49 DUB with no risk and no initial out-of-pocket investment (i.e., a pure arbitrage profit).

Arbitrageurs will pursue this opportunity, buying ABE (driving down the spot ABE/DUB exchange rate) and selling ABE forward (driving up the forward ABE/DUB exchange rate), until the interest rate parity relation is restored and arbitrage profits are no longer available.

Note that the no-arbitrage forward exchange rate is approximately proportional to the annual interest rate differential and the time period of the forward contract.



In [89]:
### Example 1yr no arb forward ABE/DUB exchange

### Borrow 1,000 DUB for one year at 3% to purchase ABE 
# and get 4,567.1 ABE. Invest the 4,567.1 ABE at the ABE rate of 5% 
# to have 1.05(4,567.1) = 4,795.45 ABE at the end of one year.
# Enter into a currency forward contract to exchange 4,795.45 
# ABE in one year at the forward rate of 4.6000 ABE/DUB 
# to receive 4,795.45 / 4.6000 = 1,042.49 DUB.

# Purchase 1000 ABE at 3%
borrowDub = round(1.05 * 4567.1,2)
print(borrowDub)

forwardDub = round(4795.45/4600,5) *1000
print("profit of the arb is =",forwardDub)

4795.46
profit of the arb is = 1042.49


#### Calculating a Forward Exchange Rate From a Forward Quote in Points or in Percentage Terms

A forward exchange rate quote typically differs from the spot quotation and is expressed in terms of the difference between the spot exchange rate and the forward exchange rate. One way to indicate this is with points. The unit of points is the last decimal place in the spot rate quote. For a spot currency quote to four decimal places, such as 2.3481, each point is 0.0001, or 1/10,000th. A quote of +18.3 points for a 90-day forward exchange rate means that the forward rate is 0.00183 greater than the spot exchange rat

#### Example: Forward exchange rates in points

The AUD/EUR spot exchange rate is 0.7313 with the 1-year forward rate quoted at +3.5 points. What is the 1-year forward AUD/EUR exchange rate?

#### Answer:

The forward exchange rate is $0.7313 + 0.00035 = 0.73165$.

#### Example: Forward exchange rates in percentage

The AUD/EUR spot exchange rate is 0.731, and the 120-day forward exchange rate is given as –0.062%. What is the 120-day forward AUD/EUR exchange rate?

#### Answer:

The forward exchange rate is 0.7313 (1 − 0.00062) = 0.7308.


In [18]:
### The AUD/EUR spot exchange rate is 0.7313 with the 
### 1-year forward rate quoted at +3.5 points. 
## What is the 1-year forward AUD/EUR exchange rate?

AUDEUR1yr = 0.7313 + 0.00035
print("-", round(AUDEUR1yr,5), "is fwd 1-year ")


### The AUD/EUR spot exchange rate is 0.731, 
### and the 120-day forward exchange rate is given as –0.062%. 
### What is the 120-day forward AUD/EUR exchange rate?

AUDEUR120d = 0.7313 * (1 - 0.00062)
print("-", round(AUDEUR120d,5), "is fwd 120-day ")

- 0.73165 is fwd 1-year 
- 0.73085 is fwd 120-day 


<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/09.png?raw=true">
<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/10.png?raw=true">

While forward rates are typically quoted as forward points, one application where percentage forward quotes are useful is if we are interpreting a forward rate as the expected future spot rate, $\large s_\text{t+1}$. We can write our no-arbitrage relation as follows:
\
\
$\Large\frac{S_t + 1}{S_t} \small- 1 = \% \Delta \large s_\text{t+1} \small = 
\Large \frac{r_\text{f} - r_d}{1+_\text{d}}$
\
\
Analysis of capital markets suggests that an increase in a country's interest rate will attract foreign investment, which will lead to appreciation of the domestic currency (spot rate decreases). This is opposite to our conclusion from our no-arbitrage forward rate result that higher domestic rates are associated with a depreciation of the domestic currency. Historically, interest rate differences are poor predictors of future spot rates, although they may be unbiased and tend to get the direction of change correct.

It's best to think of the no-arbitrage forward rate as simply the forward rate at a point in time that prevents currency arbitrage, rather than the expected future spot rate. Our no-arbitrage relation holds at a point in time and does not address the question of how changes in interest rates affect spot exchange rates over time.

#### Interpreting a Forward Discount or Premium

The **forward discount** or **forward premium** for a currency is calculated relative to the spot exchange rate. The forward discount or premium for the base currency is the percentage difference between the forward price and the spot price.

Consider the following spot and forward exchange rates:

    USD/EUR spot = $1.312

    USD/EUR 90-day forward = $1.320

The (90-day) forward premium or discount on the `euro = forward / spot − 1 = 1.320 / 1.312 − 1 = 0.610%`. Because this is positive, it is interpreted as a forward premium on the euro of `0.610%`. Because we have the forward rate for three months, we could annualize the discount simply by multiplying by `4 (= 12 / 3)`.

Because the forward quote is greater than the spot quote, it will take more dollars to buy one `euro 90 days` from now, so the euro is expected to appreciate versus the dollar, and the dollar is expected to depreciate relative to the euro.

If the forward quote were less than the spot quote, the calculated amount would be negative, and we would interpret that as a forward discount for the euro relative to the U.S. dollar.

In [102]:
###  The (90-day) forward premium or discount on the euro
fwd90EURdispremium = ((1.320 / 1.312) - 1) * 100
print("%", round(fwd90EURdispremium,4), "is fwd 90-day premium or discount on EUR ")

# NOTE: Annualized forward price

fwd360EURdispremium = fwd90EURdispremium * 4
print("%", round(fwd360EURdispremium,3), "is fwd 360-day premium or discount on EUR ")


% 0.6098 is fwd 90-day premium or discount on EUR 
% 2.439 is fwd 360-day premium or discount on EUR 


<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/11.png?raw=true">

<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/12.png?raw=true">


<hr>

# Reading 19: Key Concepts

<hr>

### LOS 19.a

Given two exchange rate quotes for three different currencies, we can calculate a currency cross rate. If the MXN/USD quote is 12.1 and the USD/EUR quote is 1.42, we can calculate the cross rate of MXN/EUR as 12.1 × 1.42 = 17.18.

### LOS 19.b

The condition that must be met so that there is no arbitrage opportunity available is as follows:

$ \Large\frac{\text{forward}}{\text{spot}} \small = \Large \frac{\big(1+i_\text{price currency}\big)}{\big(1+i_\text{base currency}\big)}$
\
\
So that:
\
\
$\text{forward} = \text{spot } \times \Large \frac{\big(1+i_\text{price currency}\big)}{\big(1+i_\text{base currency}\big)}  $
\
\
If the spot exchange rate for the euro is 1.25 USD/EUR, the euro interest rate is 4% per year, and the dollar interest rate is 3% per year, the no-arbitrage one-year forward rate can be calculated as follows:

    1.25 × (1.03 / 1.04) = 1.238 USD/EUR

Points in a foreign currency quotation are in units of the last digit of the quotation. For example, a forward quote of +25.3 when the USD/EUR spot exchange rate is 1.4158 means that the forward exchange rate is 1.4158 + 0.00253 = 1.41833 USD/EUR.

For a forward exchange rate quote given as a percentage, the percentage (change in the spot rate) is calculated as forward / spot – 1. A forward exchange rate quote of +1.787%, when the spot USD/EUR exchange rate is 1.4158, means that the forward exchange rate is 1.4158 (1 + 0.01787) = 1.4411 USD/EUR.

If a forward exchange rate does not correctly reflect the difference between the interest rates for two currencies, an arbitrage opportunity for a riskless profit exists. In this case, borrowing one currency, converting it to the other currency at the spot rate, investing the proceeds for the period, and converting the end-of-period amount back to the borrowed currency at the forward rate will produce more than enough to pay off the initial loan, with the remainder being a riskless profit on the arbitrage transaction.

To calculate a forward premium or discount for the base currency, use the following formula:

$\Large \frac{\text{forward}}{\text{spot}} \small - 1$

Forward exchange rates are poor predictors of future exchange rates. Forward exchange rates should be interpreted as simply the no-arbitrage forward rates at a point in time, not as predictors of future spot rates.

In [7]:
# Question 1

### Cross rate is (NZD/IDR) derived by (IDR/USD) divided by (NZD/USD) 
IDRNZD = (1.6000 / 2400)
print("", round(IDRNZD,5))

 0.00067


<hr>

# Module 19 Quiz

<hr>

**Question 1** \
Today's spot rate for the Indonesian rupiah (IDR) is IDR/USD 2,400.00, and the New Zealand dollar trades at NZD/USD 1.6000. The NZD/IDR cross rate is:



**Answer:** \
0.00067.



**Explanation:** \
Start with one NZD and exchange for 1 / 1.6 = 0.625 USD. Exchange the USD for 0.625 × 2,400 = 1,500 IDR. We get a cross rate of 1,500 IDR/NZD, or 1 / 1,500 = 0.00067 NZD/IDR.
\
\
(Module 19.1, LOS 19.a)

<hr>

**Question 2** \
The New Zealand dollar (NZD) is trading at USD/NZD 0.3500, and the Swedish krona (SEK) is trading at NZD/SEK 0.3100. The USD/SEK cross rate is:


**Answer:** \
0.1085.


**Explanation:** \
USD/NZD 0.3500 × NZD/SEK 0.3100 = USD/SEK 0.1085. Notice that the NZD term cancels in the multiplication. (Module 19.1, LOS 19.a)
\
\
(Module 19.1, LOS 19.a)

<hr>

**Question 3** \
The spot Swiss franc / British pound (CHF/GBP) exchange rate is 1.3050. In the 180-day forward market, the CHF/GBP exchange rate is –42.5 points. The 180-day forward CHF/GBP exchange rate is closest to:



**Answer:** \
1.3008.


**Explanation:** \
The 180-day forward exchange rate is 1.3050 – 0.00425 = CHF/GBP 1.30075.
\
\
(Module 19.1, LOS 19.b)

<hr>

**Question 4** \
The spot rate on the New Zealand dollar (NZD) is NZD/USD 1.4286, and the 180-day forward rate is NZD/USD 1.3889. This difference means:


**Answer:** \
interest rates are higher in the United States than in New Zealand.


**Explanation:** \
Interest rates are higher in the United States than in New Zealand. It takes fewer NZD to buy one USD in the forward market than in the spot market.
\
\
(Module 19.1, LOS 19.b)


<hr>

**Question 5** \
The current spot rate for the British pound (GBP) in terms of U.S. dollars is \\$1.533 and the 180-day forward rate is \\$1.508. Relative to the pound, the dollar is trading *closest* to a 180-day forward:



**Answer:** \
premium of 1.66%.


**Explanation:** \
To calculate a percentage forward premium or discount for the U.S. dollar, we use the dollar to be the base currency. The spot and forward quotes given are U.S. dollars per British pound (USD/GBP), so we must invert them to GBP/USD. The spot GBP/USD price is `1 / 1.533 = 0.6523` and the forward GBP/USD price is `1 / 1.508 = 0.6631`. Because the forward price is greater than the spot price, we say the dollar is at a forward premium of `0.6631 / 0.6523 – 1 = 1.66%`. Alternatively, we can calculate this premium with the given quotes as spot / forward – 1 to get `1.533 / 1.508 – 1 = 1.66%`.
\
\
(Module 19.1, LOS 19.a)


<hr>

**Question 6** \
The annual interest rates in the United States (USD) and Sweden (SEK) are 4% and 7% per year, respectively. If the current spot rate is SEK/USD 9.5238, then the 1-year forward rate in SEK/USD is:



**Answer:** \
9.7985.


**Explanation:** \
The forward rate in SEK/USD is $9.5238 \times \Big(\Large\frac{1.07}{1.04} \Big) \small =9.7985$. Because the SEK interest rate is the higher of the two, the SEK must depreciate approximately 3%. \
\
(Module 19.1, LOS 19.b)

<hr>

**Question 7** \
The annual risk-free interest rate is 10% in the United States (USD) and 4% in Switzerland (CHF), and the 1-year forward rate is USD/CHF 0.80. Today's USD/CHF spot rate is closest to:



**Answer:** \
0.7564.


**Explanation:** \
We can solve for the spot rate as follows. With the exchange rates quoted as USD/CHF, the spot is
$0.80 \times \Big(\Large\frac{1.04}{1.10} \Big) \small = 0.7564$ Because the interest rate is higher in the United States, it should take fewer USD to buy CHF in the spot market. In other words, the USD must be at a forward discount relative to the CHF.
\
\
(Module 19.1, LOS 19.b)

<hr>

<hr>

# Module 19 QBANK

<hr>

**Question 1** \
The spot exchange rate is 0.6243 USD/GBP and the 1-year forward rate is quoted as 3.016%. The 1-year forward exchange rate for USD/GBP is closest to:



**Answer:** \
0.6431.



**Explanation:** \
The one year forward rate is 0.6243 × (1 + 0.03016) = 0.6431.

\
\
(Module 19.1, LOS 19.b)
<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/Qbank1.png?raw=true">

<hr>

**Question 2** \
If the spot exchange rate between the British pound and the U.S. dollar is GBP/USD 0.7775, and the spot exchange rate between the Canadian dollar and the British pound is CAD/GBP 1.8325, what is the USD/CAD spot cross exchange rate?




**Answer:** \
0.70186.



**Explanation:** \
    
    First, convert GBP/USD 0.7775 to 1/0.7775 = USD/GBP 1.28617.

    Then, divide USD/GBP 1.28617 by CAD/GBP 1.8325 = USD/CAD 0.70187.
\
\
(Module 19.1, LOS 19.a)


<hr>

**Question 3** \
The Japanese yen is trading at JPY/USD 115.2200 and the Danish krone (DKK) is trading at JPY/DKK 16.4989. The USD/DKK exchange rate is:




**Answer:** \
0.1432.



**Explanation:** \
The cross rate between USD and DKK is calculated in the following manner:

    (USD/JPY)(JPY/DKK) = (1 / 115.2200) × 16.4989 = USD/DKK 0.1432 (the Yen cancels out)

\
\
(Module 19.1, LOS 19.a)

<hr>

**Question 4** \
Spot and one-month forward exchange rates are as follows:

<table>
<thead>
  <tr>
    <th></th>
    <th>Spot</th>
    <th>1-month forward</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td>EUR/DEF</td>
    <td>2.5675</td>
    <td>2.5925</td>
  </tr>
  <tr>
    <td>EUR/GHI</td>
    <td>4.3250</td>
    <td>4.2800</td>
  </tr>
  <tr>
    <td>EUR/JKL</td>
    <td>7.0625</td>
    <td>7.0075</td>
  </tr>
</tbody>
</table>

Based on these exchange rates, the EUR is closest to a 1-month forward:



**Answer:** \
premium of 1% to the GHI.



**Explanation:** \
The EUR is at a forward premium to the GHI because the EUR/GHI forward rate is less than the EUR/GHI spot rate. The base currency, GHI, is at a forward discount of forward/spot – 1 = 4.2800 / 4.3250 – 1 = –1.04%. The EUR is at a forward discount to the DEF and a forward premium to the JKL.
\
\
(Module 19.1, LOS 19.b)
<img src="https://github.com/PachaTech/CFA-Level-1/blob/main/Economics/Module%2019/pics/Qbank4.png?raw=true">